### PR TITLE
R doc 1652 snmp alerts

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+{CODE-BLOCK:json}
+{
+    ...
+    "Monitoring.Snmp.Enabled": true
+    ...
+}
+{CODE-BLOCK/}
+
+There are two configurable SNMP properties in RavenDB: the SNMP port and the community string.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: Need to add info on SNMPv3.

File: Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+It is usually easy to query the exposed metrics using a monitoring tool. ([Example](./setup-zabbix)).   
+However, you should also be able to access those directly using any SNMP agent like [Net-SNMP](http://net-snmp.sourceforge.net/).   
+
+The most basic SNMP commands are `snmpget`, `snmpset` and `snmptrapd` ([Read more](http://net-snmp.sourceforge.net/tutorial/tutorial-5/commands/)).   
+
+Each metric has a unique identifier (OID) and can be accessed individually.   
+For example, using the SNMP agent you could run the following snmpget command which gets the server up time metric:
+{CODE-BLOCK:plain}
+snmpget -v 2c -c ravendb live-test.ravendb.net 1.3.6.1.4.1.45751.1.1.1.1.1
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: explain that the first one is for SNMP protocol version 2c, second - 3

File: Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+snmpget -v 3 -l authNoPriv -u ravendb -a SHA -A ravendb live-test.ravendb.net 1.3.6.1.4.1.45751.1.1.1.2
+{CODE-BLOCK/}
+
+Where **ravendb** is the community string (configured via `Monitoring.Snmp.Community` configuration option) and "live-test.ravendb.net" is the host.
```
Comment (gregolsky) [inline Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: Describe what are these `-l authNoPriv -u ravendb -a SHA -A ravendb`

File: Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: Done

File: Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: Done

File: Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown (hunk 3 above)
Comment (morhilai) [reply Documentation/4.2/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: Done

**Pull request merged**